snuba/datasets/entities/sessions.py

```diff
@@ -266,3 +270,34 @@ def get_query_processors(self) -> Sequence[QueryProcessor]:
             ),
             ProjectRateLimiterProcessor(project_column="project_id"),
         ]
+
+
+class OrgSessionsEntity(Entity):
+    def __init__(self) -> None:
+        storage = get_storage(StorageKey.SESSIONS_HOURLY)
+
+        super().__init__(
+            storages=[storage],
+            query_pipeline_builder=SimplePipelineBuilder(
+                query_plan_builder=SingleStorageQueryPlanBuilder(storage=storage)
+            ),
+            abstract_column_set=ColumnSet(
+                [
+                    ("org_id", UInt(64)),
+                    ("project_id", UInt(64)),
+                    ("started", DateTime()),
+                ]
+            ),
+            join_relationships={},
+            writable_storage=None,
+            validators=None,
+            required_time_column=None,
```

Comment:
Mentioned this in Slack, but you should still have a required time column. You'll have to add back the TimeSeriesProcessor.